android/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,20 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.bleplx">
+          package="com.bleplx"
+          xmlns:tools="http://schemas.android.com/tools">
+
+  <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+  <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
Location is no longer needed for Android 12+, so it could be annotated with maxSdkVersion (31?) even though the maxSdkVersion actually doesn't do anything. It will end up in all apps, and just suppresses Android Studio warnings.

Reply:
I Agree with you. 
As I wrote in previous comment - there's a lot to cleanup in this codebase, and we are slowly taking care of issues and improvements here.